lib/BiochemistryAPI/utils.py

```diff
@@ -52,20 +53,20 @@ def dict_from_file(path, key='id', dialect='excel-tab'):
         id_dict[line[key]] = line
         for tok in _tokenize_string(line[key]):
             alias_dict[tok].add(line[key])
-        for tok in _tokenize_string(line.get('name')):
+        for tok in _tokenize_string(line.get("name")):
             alias_dict[tok].add(line[key])
-        for tok in _tokenize_string(line.get('abbreviation')):
+        for tok in _tokenize_string(line.get("abbreviation")):
             alias_dict[tok].add(line[key])
-        for tok in _tokenize_string(line.get('ec_number')):
+        for tok in _tokenize_string(line.get("ec_number")):
             alias_dict[tok].add(line[key])
-        if line.get('aliases'):
-            for match in re.findall('"\S+?:(\S+?)"', line['aliases']):
+        if line.get("aliases"):
+            for match in re.findall('"\S+?:(\S+?)"', line["aliases"]):
```

Comment:
This is an example `line["aliases"]` from `data/compounds.tsv`
```
In [104]: line["aliases"]                                                       
Out[104]: '["Name: ATP; Adenosine 5\'-triphosphate; adenosine-5\'-triphosphate; adenosine-triphosphate; adenylpyrophosphate", \'AraCyc: ATP\', \'BiGG: atp\', \'BrachyCyc: ATP\', \'KEGG: C00002\', \'MetaCyc: ATP\']'
```
I don't think there would ever be any match because (A) the regex pattern doesn't have a space between the colon and the succeeding token, and (B) some of the alias pairs are enclosed as strings with `'` instead of `"`. This part may need some TLC.

I might try something like
```
In [106]: eval(line["aliases"])                                                 
Out[106]: 
["Name: ATP; Adenosine 5'-triphosphate; adenosine-5'-triphosphate; adenosine-triphosphate; adenylpyrophosphate",
 'AraCyc: ATP',
 'BiGG: atp',
 'BrachyCyc: ATP',
 'KEGG: C00002',
 'MetaCyc: ATP']
```
to parse out the alias pair strings into a list (and then feed them to `_tokenize_string` I guess)

I hope I'm reading the intent right here ... if not disregard

Reply:
I'll have to double-check the intent here

Reply:
This is intentionally over-written here:
https://github.com/kbaseapps/BiochemistryAPI/blob/master/lib/BiochemistryAPI/BiochemistryAPIImpl.py#L95
and here:
https://github.com/kbaseapps/BiochemistryAPI/blob/master/lib/BiochemistryAPI/BiochemistryAPIImpl.py#L141

So I'm going to remove that block of code to avoid confusion.